### DISCUSSION TITLE
Adding Polka and Polkadot to Web Frameworks?

### DISCUSSION THREAD

**Discussion by kerim-unmind** [sindresorhus/awesome-nodejs#1098] [Ideas]
Hey! 

Just wanted to suggest perhaps adding [Polka](https://github.com/lukeed/polka) and [Polkadot](https://github.com/lukeed/polkadot) to the Web Frameworks section on here? 

I've been using them both now as alternatives to Micro and Express for a while now and find them robust and easy to use, considering the similar APIs. They're production-ready and I've seen even some frameworks now shipping with them as Express alternatives (Sapper, for example).

**Comment by kerim-unmind** [comment]
Realised probably better to make a PR than put this here. Sorry, but I can't seem to delete it.

**Comment by Preacher27** [comment]
[]()****
